cli.py

```diff
@@ -49,27 +57,28 @@
 @click.option("--stale_records_removal_interval", default=900,
               help="Interval period between removal of stale modification records.",
               type=int)
-@click.argument("serve", required=True)
-def startserver(adduser: Tuple, api: str, auth: bool, dburl: str, pagination: bool,
-                hydradoc: str, port: int, pagesize: int, serverurl: str, token: bool,
-                stale_records_removal_interval: int, serve: None) -> None:
+def serve(adduser: tuple, api: str, auth: bool, dburl: str, pagination: bool,
+          hydradoc: str, port: int, pagesize: int, serverurl: str, token: bool,
+          stale_records_removal_interval: int) -> None:
     """
-    Python Hydrus CLI
+    Starts up the server.
+    \f
```

Comment:
please remove this line

Reply:
The line 65 would help to keep the help mode more clean for the user. We are already providing with the types of input parameter in help mode so that would be unnecessary replication.What do you think?